src/containerapp/azext_containerapp/_clients.py

```diff
@@ -1382,3 +1383,98 @@ def list(cls, cmd, resource_group_name, environment_name):
             dotNet_component_list.append(component)
 
         return dotNet_component_list
+
+
+class MaintenanceConfigPreviewClient():
+    api_version = "2024-10-02-preview"
```

Comment:
Please sync main branch.
With PR: https://github.com/Azure/azure-cli-extensions/pull/8222
I have updated api-version to 2024-10-02-preview

Reply:
updated PREVIEW_API_VERSION to "2024-10-02-preview" and use PREVIEW_API_VERSION as pr is not merged yet

Reply:
@p-bouchon It has been merged now, please resolve the conflict and ensure the CI pass